migrations/0046-add-food-column.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+const TABLE_NAME = 'Events';
+
+export class AddFoodColumn1728959627663 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.addColumns(TABLE_NAME, [
```

Comment:
Syntactical nit: I don't think we need `[]` if you're just adding a single column